frontend/packages/data-portal/app/components/Layout/TopNavigation.tsx

```diff
@@ -1,15 +1,40 @@
 import { useLocation } from '@remix-run/react'
+import { useTranslation } from 'react-i18next'
 
 import { Link } from 'app/components/Link'
-import { i18n } from 'app/i18n'
+import { I18nKeys } from 'app/types/i18n'
 import { cns } from 'app/utils/cns'
+import { useFeatureFlag } from 'app/utils/featureFlags'
 
 import { AboutAndHelpDropdown } from './AboutAndHelpDropdown'
 import { CryoETHomeLink } from './CryoETHomeLink'
 import { ToolsDropdown } from './ToolsDropdown'
 
+interface TopNavLink {
+  isActive(pathname: string): boolean
+  label: I18nKeys
+  link: string
+}
+
+const TOP_NAV_LINKS: TopNavLink[] = [
+  {
+    isActive: (pathname) =>
+      pathname.includes('/datasets') || pathname.includes('/runs'),
```

Comment:
General, non blocking, out of scope question: 

In Cell Science, we use a `routes.ts` file to hold route constants, so we have single source of truth, like so:

```ts
export enum ROUTES {
  HOMEPAGE = "/",
  CENSUS_DIRECTORY = "/census-models",
  COLLECTION = "/collections/:id",
  COLLECTIONS = "/collections",
  ...
}
```

Is it something worth doing here as a separate PR 👌 ?